test/soca/gw/CMakeLists.txt

```diff
@@ -50,7 +50,7 @@ set(jjob_list "JGLOBAL_PREP_OCEAN_OBS"
               "JGDAS_GLOBAL_OCEAN_ANALYSIS_BMAT"
               "JGDAS_GLOBAL_OCEAN_ANALYSIS_RUN"
               "JGDAS_GLOBAL_OCEAN_ANALYSIS_ECEN"
-#	            "JGDAS_GLOBAL_OCEAN_ANALYSIS_LETKF"
+	      "JGDAS_GLOBAL_OCEAN_ANALYSIS_LETKF"
```

Comment:
would that test run without failing with the current g-w develop?

Reply:
Without the half-done run method, yes. Shall I put it in?